anta/tests/interfaces.py

```diff
@@ -191,16 +192,19 @@ def test(self) -> None:
 
 
 class VerifyInterfacesStatus(AntaTest):
-    """Verifies if the provided list of interfaces are all in the expected state.
+    """Validates whether the given interfaces are in their expected operational states.
 
-    - If line protocol status is provided, prioritize checking against both status and line protocol status
-    - If line protocol status is not provided and interface status is "up", expect both status and line protocol to be "up"
-    - If interface status is not "up", check only the interface status without considering line protocol status
+    This test performs the following checks for each specified interface:
```

Comment:
![Screenshot 2024-10-28 at 4 00 35 PM](https://github.com/user-attachments/assets/78d1db1e-fbe1-4748-a9bc-839d58eb52fc)
This format will not reflect properly in documentation. Please change it  with hyphen

Reply:
Updated.

Reply:
You can use numbered list but you need to put an extra indent.

Reply:
Updated.
<img width="1111" alt="Screenshot 2024-10-30 at 10 20 08 AM" src="https://github.com/user-attachments/assets/b8f54c7c-5bb4-4a6b-81b2-231798f60f03">